mods/ts/rules/civilian-structures.yaml

```diff
@@ -1254,14 +1254,13 @@ GALITE:
 	-WithMakeAnimation:
 	-WithDeathAnimation:
 	Health:
-		HP: 600
+		HP: 60000
 	Armor:
 		Type: Wood
 	RevealsShroud:
 		Range: 2c0
 	Power:
 		Amount: 0
-# TODO: should use terrain lighting instead, depends on https://github.com/OpenRA/OpenRA/issues/3605
```

Comment:
The diff was too large for my gui to show this file completely, so I couldn't discard this change. I'll restore it before this gets merged.